.eslintrc.json

```diff
@@ -2,6 +2,7 @@
     "env": {
         "commonjs": true,
         "shared-node-browser": true,
+        "browser": true,
```

Comment:
I don't want this on, as we also support Node.  Each use of a browser global should have to be manually toggled per file with eslint comments.